lib/client/client-spec.js

```diff
@@ -0,0 +1,79 @@
+var mediator = require('fh-wfm-mediator/lib/mediator');
+var sinon = require('sinon');
+var assert = require('assert');
+
+describe("Raincatcher Sync Client Initialisation", function() {
+
+  var mockSyncOptions = {
+    "sync_frequency": 5,
+    "storage_strategy": "dom",
+    "do_console_log": false
+  };
+
+  var mockDataSetId = "mockdatasetid";
+
+  var mockDataSetFilterParams = {filterParam1: "filtervalue"};
+  var mockDataSetMetadataParams = {metadataParam1: "metadatavalue"};
+
+  before(function() {
+    var self = this;
+
+    //Clearing the require cache to re-require the client
+    self.syncClient = require('../client');
```

Comment:
No real benefit as the require is a singleton anyway. I will move to the describe.